src/snowflake/snowpark/modin/pandas/series_utils.py

```diff
@@ -101,572 +78,3 @@ def as_ordered(self, inplace=False):
-        return Series(query_compiler=self._query_compiler.dt_property("time"))
-
-    @property
-    def timetz(self):
-        return Series(query_compiler=self._query_compiler.dt_property("timetz"))
-
-    @property
-    def year(self):
-        return Series(query_compiler=self._query_compiler.dt_property("year"))
-
-    @property
-    def month(self):
-        return Series(query_compiler=self._query_compiler.dt_property("month"))
-
-    @property
-    def day(self):
-        return Series(query_compiler=self._query_compiler.dt_property("day"))
-
-    @property
-    def hour(self):
-        return Series(query_compiler=self._query_compiler.dt_property("hour"))
-
-    @property
-    def minute(self):
-        return Series(query_compiler=self._query_compiler.dt_property("minute"))
-
-    @property
-    def second(self):
-        return Series(query_compiler=self._query_compiler.dt_property("second"))
-
-    @property
-    def microsecond(self):
-        return Series(query_compiler=self._query_compiler.dt_property("microsecond"))
-
-    @property
-    def nanosecond(self):
-        return Series(query_compiler=self._query_compiler.dt_property("nanosecond"))
-
-    @property
-    def dayofweek(self):
-        return Series(query_compiler=self._query_compiler.dt_property("dayofweek"))
-
-    @property
-    def weekday(self):
-        return Series(query_compiler=self._query_compiler.dt_property("weekday"))
-
-    @property
-    def dayofyear(self):
-        return Series(query_compiler=self._query_compiler.dt_property("dayofyear"))
-
-    @property
-    def quarter(self):
-        return Series(query_compiler=self._query_compiler.dt_property("quarter"))
-
-    @property
-    def is_month_start(self):
-        return Series(query_compiler=self._query_compiler.dt_property("is_month_start"))
-
-    @property
-    def is_month_end(self):
-        return Series(query_compiler=self._query_compiler.dt_property("is_month_end"))
-
-    @property
-    def is_quarter_start(self):
-        return Series(
-            query_compiler=self._query_compiler.dt_property("is_quarter_start")
-        )
-
-    @property
-    def is_quarter_end(self):
-        return Series(query_compiler=self._query_compiler.dt_property("is_quarter_end"))
-
-    @property
-    def is_year_start(self):
-        return Series(query_compiler=self._query_compiler.dt_property("is_year_start"))
-
-    @property
-    def is_year_end(self):
-        return Series(query_compiler=self._query_compiler.dt_property("is_year_end"))
-
-    @property
-    def is_leap_year(self):
-        return Series(query_compiler=self._query_compiler.dt_property("is_leap_year"))
-
-    @property
-    def daysinmonth(self):
-        return Series(query_compiler=self._query_compiler.dt_property("daysinmonth"))
-
-    @property
-    def days_in_month(self):
-        return Series(query_compiler=self._query_compiler.dt_property("days_in_month"))
-
-    @property
-    def tz(self) -> "tzinfo | None":
```

Comment:
Some of the methods like `tz` here have an implementation - why not keep the implementation? Also if you are removing it, can you also update the documentation/supported methods rst files? Thanks!

Reply:
We still support the method, we're just using OSS modin's `DatetimeProperties` class for the frontend instead. Unlike `StringMethods`, our existing non-implementation for most datetime methods error at the query compiler layer instead of the frontend.

https://github.com/modin-project/modin/blob/31771d7bb9d1dc36fb26ec49dc7798ccfea366b6/modin/pandas/series_utils.py#L696C5-L696C11

Reply:
Ah I see, that makes sense. Thanks!